frontend/src/components/pick-track/index.tsx

```diff
@@ -17,12 +20,8 @@ class PickTrack extends React.Component<{}, IPickTrackState> {
     return (
       <section className="pick-track u-content-wrapper">
         <div className="pick-track__content u-content">
-          <h3 className="pick-track__title u-title"> Pick your track </h3>
-          <p className="pick-track__description"> 
-            Our teams are »cross-functional«. This means that Product Managers, Interaction Designers,
-            Software Engineers and Artificial Intelligence Engineers work together to find out what the 
-            user needs and to build the product.
-          </p>
+          <h3 className="pick-track__title u-title"> { this.renderTitle() } </h3>
+          { this.renderDescription() }
```

Comment:
convention to better differentiate `{}` as `js content inside` and `{}` as object:

// { } as js content inside - no spacing
{[1,2,3].map(item => item * 2)}

/// { } as object - spacing
{ name: 'Michael' } 

Combined:

<MyComponent person={{ name: 'Michael' }} />